parser/parse.go

```diff
@@ -227,9 +227,16 @@ func (b *Builder) AddDirRecursive(dir string) error {
 		klog.Warningf("Ignoring directory %v: %v", dir, err)
 	}
 
+	// filepath.Walk does not follow symlinks. We therefore evaluate symlinks and use that with
+	// filepath.Walk.
+	realPath, err := filepath.EvalSymlinks(b.buildPackages[dir].Dir)
+	if err != nil {
+		return err
+	}
+
 	// filepath.Walk includes the root dir, but we already did that, so we'll
 	// remove that prefix and rebuild a package import path.
-	prefix := b.buildPackages[dir].Dir
+	prefix := realPath
```

Comment:
I see this was also in the original code but what's the purpose of this variable?

Reply:
It's used in the following block of code, but I've removed it and used `realPath` there.